addons/armory/stringtable.xml

```diff
@@ -114,6 +115,7 @@
         </Key>
         <Key ID="STR_TAC_Armory_ShowOnlyCompatible">
             <English>Show only items compatible with equipped weapons.</English>
+            <German>Zeige nur Gegenstände, die mit ausgerüsteten Waffen kompatibel sind</German>
```

Comment:
Now there is a missing point at the end which was under Italian before. :laughing: